app/models/post_mover.rb

```diff
@@ -224,6 +224,7 @@ def posts
     @posts ||= begin
       Post.where(topic: @original_topic, id: post_ids)
         .where.not(post_type: Post.types[:small_action])
+        .where.not(raw: '')
```

Comment:
This is quite a code smell :(

Again like the other PR I approved it’s not your fault, but we really should figure out a cleaner way to do this in the future.